src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -1,20 +1,26 @@
 import struct as st
+from typing import Optional, Tuple
+
+import asyncserial
 
 from thorlabs_cube.driver.base import _Cube
 from thorlabs_cube.driver.message import MGMSG, Message, MsgError
 
+_CHANNEL = 0x01
+_RESERVED = 0x00
+
 
 class Tpz(_Cube):
     """Either :py:meth:`set_tpz_io_settings()<Tpz.set_tpz_io_settings>`
     or :py:meth:`get_tpz_io_settings()<Tpz.get_tpz_io_settings>` must
     be completed to finish initialising the driver.
     """
 
-    def __init__(self, loop, serial_dev):
-        _Cube.__init__(self, loop, serial_dev)
-        self.voltage_limit = None
+    def __init__(self, serial_dev) -> None:
+        self.port = asyncserial.AsyncSerial(serial_dev, baudrate=115200, rtscts=True)
+        self.voltage_limit: Optional[int] = None
```

Comment:
How does this attribute get set? [Attributes cannot be queried or set with sipyco](https://github.com/m-labs/sipyco/issues/40#issuecomment-2399006930), so may need to add getter/setter methods.

Reply:
As far as I understand the issue you suggested with this, the voltage limit is set with the `set/get tpz_io_settings` function

Reply:
Set in `get/set_tpz_io_settings` method.